tests/test_class_optimize.py

```diff
@@ -64,10 +64,11 @@ def test_optimize(
 
     # Call the optimization function
     ergebnis = opt_class.optimierung_ems(parameters=input_data, start_hour=start_hour, ngen=ngen)
+    # from akkudoktoreos.class_numpy_encoder import NumpyEncoder
+
     # with open(f"new_{fn_out}", "w") as f_out:
-    #     from akkudoktoreos.class_numpy_encoder import NumpyEncoder
```

Comment:
probably a merge/rebase artifact, however here you probably want to export the parameters as well so they can be used as visualization test input.